src/general/MainMenu.tscn

```diff
@@ -417,6 +417,7 @@ text = "BACK"
 
 [node name="OptionsMenu" parent="." instance=ExtResource( 3 )]
 visible = false
+DisplayPartNamesTogglePath = null
```

Comment:
This seems a bit suspicious change...

Reply:
Uh oh, I must have accidently commited godot's changes